driver-core/src/test/java/com/datastax/driver/core/CCMBridge.java

```diff
@@ -291,87 +285,57 @@ public static Builder builder() {
         return new Builder();
     }
 
-    /**
-     * Returns the address of the {@code nth} host in the CCM cluster (counting from 1, i.e.,
-     * {@code getHostAddress(1)} returns the address of the first node.
-     * <p/>
-     * In multi-DC setups, nodes are numbered in ascending order of their datacenter number.
-     * E.g. with 2 DCs and 3 nodes in each DC, the first node in DC 2 is number 4.
-     *
-     * @return the address of the {@code nth} host in the cluster.
-     */
-    public InetSocketAddress addressOfNode(int n) {
-        return new InetSocketAddress(TestUtils.ipOfNode(n), binaryPort);
+    @Override
+    public String getClusterName() {
+        return clusterName;
     }
 
-    /**
-     * Returns an {@link AddressTranslater} to use when tests must manually create
-     * {@link Cluster} instances.
-     * <p/>
-     * This translator is necessary because hosts in this cluster do not necessarily use standard ports.
-     *
-     * @return an {@link AddressTranslater} to use when tests must manually create
-     * {@link Cluster} instances.
-     */
-    public AddressTranslater addressTranslator() {
```

Comment:
I noticed a bit too late that Cluster.Builder addContactPoints(0 + withPorts() is preferable compared to addContactPointsWithPorts() + withAddressTranslator(), so I deleted this method altogether.